RELEASING.md

```diff
@@ -1,14 +1,45 @@
-1. Update the version number in `RCPurchases.m`
-1. Update the version number in `Purchases.podspec` and in `Purchases/Info.plist` by running `fastlane bump version:x.y.z`
+#### (Somewhat) automated process: 
+1. Create a branch bump/x.y.z
+1. Update the version number in `RCPurchases.m`, `Purchases.podspec` and in 
+`Purchases/Info.plist` by running `fastlane bump version:x.y.z`
 1. Update CHANGELOG.md for the new release
 1. Commit the changes `git commit -am "Version x.y.z"`
+1. Make a PR, merge when approved
+1. `cd bin`
+1. `./release_version.sh -c x.y.z -n a.b.c`, where a.b.c will be the next release after this one. 
+If you're releasing version 3.0.2, for example, this would be `./release_version.sh -c 3.0.2 -n 3.1.0`. 
+This will do all of the other steps in the manual process.
+1. Create a [new github release](https://github.com/revenuecat/purchases-ios/releases)
+1. Upload the contents of `CarthageUploads/` to the new release.   
+
+#### Manual process:
+
+1. Create a branch bump/x.y.z
+1. Update the version number in `RCPurchases.m`, `Purchases.podspec` and in 
+`Purchases/Info.plist` by running `fastlane bump version:x.y.z`
+1. Update CHANGELOG.md for the new release
+1. Commit the changes `git commit -am "Version x.y.z"`
+1. Make a PR, merge when approved
 1. `git tag -a x.y.z -m "Version x.y.z"`
-1. `git push origin master && git push --tags`
+1. `git push origin bump/x.y.z && git push --tags`
```

Comment:
Maybe I am missing something but there is a step before this that says to make a PR. Shouldn't the push to to bum/x.y.z happen before opening the PR? Also, shouldn't the tagging be on `master`?

Reply:
Nevermind I understand now. It's for the SNAPSHOT change. Then the push in line 45 should be to this branch right? not to master :)